easybuild/easyconfigs/r/ROOT/ROOT-v6.10.08-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'CMakeMake'
```

Comment:
@Helios07 Why not use the existing easyblock for `ROOT`?

I.e., just remove this line, and let `eb` search for the `EB_ROOT` easyblock in `root.py` (see https://github.com/easybuilders/easybuild-easyblocks/blob/master/easybuild/easyblocks/r/root.py).